interfaces/builtin/vulkan_driver_libs.go

```diff
@@ -156,11 +156,11 @@ func checkVulkanApiVersion(slot *interfaces.ConnectedSlot, apiVersion string) er
 	if len(version) < 2 {
 		return fmt.Errorf("api_version is not a version: %s", apiVersion)
 	}
-	major, err := strconv.Atoi(version[0])
+	major, err := strconv.ParseUint(version[0], 10, 64)
 	if err != nil {
 		return fmt.Errorf("while parsing major: api_version %s", apiVersion)
```

Comment:
sorry, I didn't notice but there's no error definition in this? usually we use while if there's a %w or something,  this needs "invalid" in there perhaps?

Reply:
that's an odd error message, I still don't think is very clear:

invalid icd-source: nvidia.json:  invalid major number: api_version 1x.1

maybe it's better,  it's a case where trying to avoid the double "invalid" just muddies things